apps/bfd-data-npi/build.sh

```diff
@@ -0,0 +1,5 @@
+mvn --quiet clean install
+
+mvn --quiet exec:java \
+-Dexec.mainClass="gov.cms.bfd.data.npi.utility.App" \
+-Dexec.args="./src/main/resources/"
```

Comment:
lets remove this script and instead add support via system property to the maven build for optionally providing the location of the NPI file that is to be used. for now, keep the java fallback but lets revisit that once the jenkins workflow is functioning properly